### PR TITLE
Documentation for phase and span comparers

### PR DESCRIPTION
**Pull request by jolyonb**
This is documentation for #175. Also updates python_lib.zip.

### CODE REVIEW

File: course/problem/matrix7.xml [hunk 1]
```diff
@@ -1,49 +1,65 @@
...
+</script>
+
+<p>
+<p style="display:inline">\(\vec{v} = \) </p>
+<customresponse cfn="grader2" expect="[1, i]" inline="1">
+  <textline math="true" preprocessorClassName="MJxPrep" preprocessorSrc="/static/MJxPrep.js" size="40" inline="1" />
 </customresponse>
+</p>
 
 <h3>Resources</h3>
```
Comment (ChristopherChudzicki) [inline course/problem/matrix7.xml]: Resource links to specify_domain docs, should go to `https://mitodl.github.io/mitx-grading-library/grading_math/comparer_functions`

File: docs/grading_math/comparer_functions.md [hunk 2]
```diff
@@ -33,9 +33,12 @@ The table below lists the comparers are exported by `mitxgraders.comparers` alon
...
+| `congruence_comparer` |  `FormulaGrader` <br/> `NumericalGrader` | `[expected, modulus]` | reduces student input modulo modulus, then checks for equality within grader's tolerance. |
+| `between_comparer` |  `FormulaGrader` <br/> `NumericalGrader` | `[start, stop]` | checks that student input is real and between `start` and `stop`.  |
+| `eigenvector_comparer` | `MatrixGrader`  | `[matrix, eigenvalue]`  | checks that student input is an eigenvector of `matrix` with eigenvalue `eigenvalue` within grader's tolerance.  |
+| `vector_phase_comparer` | `MatrixGrader`  | `[comparison_vector]`  | checks that student input is equal to `comparison_vector` up to a phase, within grader's tolerance. |
+| `vector_span_comparer` | `MatrixGrader`  | `[vector1, vector2, ...]`  | checks that student input is in the span of the given list of vectors, within grader's tolerance. If only a single vector is given, checks if the student input is equal to the given vector up to a (possibly complex) constant of proportionality.  |
```
Comment (ChristopherChudzicki) [inline docs/grading_math/comparer_functions.md]: Maybe specify nonzero: "...nonzero and in the span of...".

**Comment by jolyonb** [comment]
Hmm. You merged too fast. That's ok, I've got a new tiny little PR incoming.

**Comment by ChristopherChudzicki** [comment]
> Hmm. You merged too fast. That's ok, I've got a new tiny little PR incoming.

Yikes, sorry about that. I'm on a new computer and I accidentally added `https://github.com/mitodl/mitx-grading-library` as my origin remote instead of `https://github.com/ChristopherChudzicki/mitx-grading-library`. Fixed my remotes.

**Pull request merged**